src/index.ts

```diff
@@ -360,50 +290,18 @@ export default class Mnoga {
     // If options has a locale, normalize it and use that instead of locale. Otherwise, check
     // if locale is even set and add that to possible locales.
     if (options.locale !== undefined) {
-      locales.push(this.normalizeLocale(options.locale));
+      locales.push(getCanonicalLocale(options.locale));
     } else if (this.locale !== undefined) {
       locales.push(this.locale);
     }
 
     if (options.fallback !== undefined) {
-      locales.push(this.normalizeLocale(options.fallback));
+      locales.push(getCanonicalLocale(options.fallback));
     } else {
       locales.push(this.fallback);
     }
 
-    // Loop through the locales and try to find an appropriate translation.
-    for (let i = 0; i < locales.length; i++) {
-      const locale = locales[i];
-      const contexts = key.split('.');
-
-      let translation: Translations[string] = this.translations[locale];
-
-      // Iterate through the contexts, getting the next value in the translations.
-      for (let j = 0; j < contexts.length; j++) {
-        const context = contexts[j];
-        translation = this.isTranslations(translation) ? translation[context] : undefined;
-      }
-
-      // If there is a count and an object comes back, check if this key is a plural context.
-      if (this.isTranslations(translation) && typeof data.count === 'number') {
-        const pluralContext: string = this.getPluralContext(locale, data.count);
-        translation = translation[pluralContext];
-      }
-
-      // If the translation is equal to string, interpolate and return.
-      if (typeof translation === 'string') {
-        return translation
-          .replace(INTERPOLATION_REGEX, (exp, arg) => {
-            const result = data[arg];
-            return typeof result === 'string' ? result :
-                   typeof result === 'number' ? `${result}` :
-                   exp;
-          });
-      }
-    }
-
-    // Fallback to the key.
-    return key;
+    return t({ data, key, locales, rules: this.rules, translations: this.translations });
```

Comment:
What do you think of renaming the `t` utility to something more description, like `getTranslation` or `lookupKey` or something like that? I think it's a little bit confusing to have two different methods called `t` with slightly different signatures and slightly different behavior, and the `t` utility is probably always going to be wrapped instead of being used directly, so it doesn't really need to be abbreviated.

Reply:
`t` is a super conventional name, so I kind of prefer it. Although I guess for the utility we could call it something like `getTranslation` or `lookupTranslation`. I wonder if we should change the naming convention from `translation` to `phrase`, though?